Dockerfile

```diff
@@ -2,7 +2,9 @@
 #
 FROM openshift/origin-release:golang-1.10
 COPY . /go/src/github.com/openshift/cluster-kube-apiserver-operator
-RUN cd /go/src/github.com/openshift/cluster-kube-apiserver-operator && go build ./cmd/cluster-kube-apiserver-operator
+WORKDIR /go/src/github.com/openshift/cluster-kube-apiserver-operator
+ENV GO_PACKAGE github.com/openshift/cluster-kube-apiserver-operator
+RUN go build -ldflags "-X $GO_PACKAGE/pkg/version.versionFromGit=$(git describe --long --tags --abbrev=7 --match 'v[0-9]*')" ./cmd/cluster-kube-apiserver-operator
```

Comment:
@sttts @deads2k i guess we only care about having the version set to actual version in the image we build?

Reply:
Instead of?

Reply:
instead of having the version set in make build